.github/scripts/Rename_Photos

```diff
@@ -0,0 +1,26 @@
+#This script renames photos from BHVP_PH_CetaitParis_DP_1538_01_015 and BHVP_PH_CetaitParis_DP_1538_01_016.jpg to 16_slide.jpg and 16_photo.jpg
```

Comment:
Hey — remember that this file needs to live in the scripts directory inside of our project — not .github folder